salt/scripts.py

```diff
@@ -46,9 +46,24 @@ def _handle_signals(client, signum, sigframe):
         hardcrash = client.options.hard_crash
     except (AttributeError, KeyError):
         hardcrash = False
+
+    exit_msg = '\nExiting gracefully on Ctrl-c'
+    try:
+        jid = client.local_client.pub_data['jid']
```

Comment:
What possible exceptions am I missing in the try/except, then? Would it be better to change it to an `except Exception` ?